source/help/apps/desktop-changelog.rst

```diff
@@ -1,6 +1,96 @@
+- Added support for exposing Webview Developer Tools via View Menu.
+- Improved the styling of the session expiry mention badge in the tab bar.
+- Improved the wording of the invalid certificate dialog.
+- Improved accessibility support for the menu bar items. This was fixed with the major version upgrade of Electron to v5.0.0.
+
+Windows
+^^^^^^^^^^^^^
+
+- Added support for MSI installer (Beta) to allow deploying Mattermost desktop app to the computer program files (accessible by any user accounts rather than a specific user account on the machine).
+- Added support for Group Policies (GPO) to allow admins to set default servers and enable/disable the ability to add/remove servers.
+
+Mac
+^^^^^^^^^^^^^
+
+- Added a flag to enable MacOS dark mode title bar.
+
+Architectural Changes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- Major version upgrade of Electron to v5.0.0. Electron is the underlying technology used to build the Desktop apps.
+
+Bug Fixes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+All Platforms
+^^^^^^^^^^^^^
+
+- Fixed an issue where opening the emoji picker froze the desktop app. This bug was fixed with the major version upgrade of Electron to v5.0.0.
+- Fixed an issue where jumbo emoji didn't render for unsupported unicode emojis. This bug was fixed with the major version upgrade of Electron to v4.0.0.
+- Fixed an issue where username and password were not being passed for HTTP basic authentication.
+- Fixed an issue where switching server tabs on app load caused a visual size glitch. This bug was fixed with the major version upgrade of Electron to v4.0.0.
+- Fixed various desktop app notification issues.
+- Fixed an issue where the unread count changed after opening the quick switcher.
+- Fixed an issue where clicking on some links in System Console opened the links on the app itself.
+- Fixed an issue where the "Help" button opened in a new browser tab instead of below the textbox in the default system browser.
+- Fixed an issue where Mattermost opened both on fullscreen and on a smaller window when closing the app in fullscreen.
+- Fixed an issue to prevent the app from restarting in full-screen mode.
+- Fixed an issue where the dot and mention counts in server tab jewels were not centered.
+- Fixed an issue where the dot in notification badges was off centre.
+
+Windows
+^^^^^^^^^^^^^
+
+- Fixed an issue where Ctrl+M shortcut minimized the Windows app and sent a message.
+- Fixed an issue where clicking the tooltip button dismissed the tooltip.
+
+Mac
+^^^^^^^^^^^^^
+
+- Fixed an issue where using the red Close button to close the window caused a blank screen when the window was maximized.
+- Fixed an issue where ``Cmd + Option + Shift + v`` and ``Cmd + Shift + v`` didn't work on MacOS desktop app. This bug was fixed with the major version upgrade of Electron to v5.0.0.
+- Fixed an issue where the timezones were incorrect in OSX High Sierra. This bug was fixed with the major version upgrade of Electron to v4.0.0.
+
+Known Issues
```

Comment:
@esethna proposed moving this higher and included some tweaks to the changelog on the Desktop repo, should we apply here as well?

Reply:
Made the updates.